docs/BR.md

```diff
@@ -1368,7 +1384,7 @@ Revocation entries on a CRL or OCSP Response MUST NOT be removed until after the
 
 ### 4.10.2 Service availability
 
-The CA SHALL operate and maintain its CRL and OCSP capability with resources sufficient to provide a response time of ten seconds or less under normal operating conditions.
+The CA SHALL operate and maintain its CRL and optional OCSP capability with resources sufficient to provide a response time of ten seconds or less under normal operating conditions.
```

Comment:
In #402, @BenWilson-Mozilla commented:

I think there has been discussion about eliminating or re-phrasing this requirement.

[will leave open for community discussion]

Reply:
Closing stale comments.